problem_builder/answer.py

```diff
@@ -178,6 +178,8 @@ def mentoring_view(self, context=None):
         """ Render this XBlock within a mentoring block. """
         context = context.copy() if context else {}
         context['self'] = self
+        context['label_correct'] = self._(u'Correct answer')
+        context['label_incorrect'] = self._(u'Incorrect answer')
```

Comment:
Could we embed these strings directly in the templates (`data-label_correct="{% trans "Correct answer" %}"`, etc)?